posts/2021/05/hpy-irc-moves-to-libera-chat.md

```diff
@@ -0,0 +1,27 @@
+<!--
+.. title: #hpy IRC moves to Libera.Chat
+.. slug: hpy-irc-moves-to-libera-chat
+.. date: 2021-05-29 10:00:00 UTC
+.. author: antocuni
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+Because of the
+[recent news about Freenode](https://arstechnica.com/gadgets/2021/05/freenode-irc-has-been-taken-over-by-the-crown-prince-of-korea/)
+and following the example of many other FOSS projects, the HPy team has
```

Comment:
I would say either add a less inflammatory link or just remove the link:
```suggestion
Following the example of many other FOSS projects, the HPy team has
```

Reply:
I'm +1 on just removing the link and stating as a fact that we're moving.